ToDo.Entities/TodoTask.cs

```diff
@@ -5,6 +5,9 @@ namespace ToDo.Entities
     public class TodoTask : IUnique
     {
         public Guid Id { get; set; }
+        public bool IsCompleted { get; set; }
+        public string Name { get; set; }
         public string Description { get; set; }
+        public DateTime? CompletionDate { get; set; }
```

Comment:
IIRC, Peter didn't really like the nullable DateTime. (Correct me if I'm wrong, @petrspelos) 
I don't mind it, however, you could also use its minimum value for Todos that don't have a date.

Reply:
@svr333 Nullable in this context makes more sense to me. Default DateTime is a bit of an anti-pattern if the DateTime can genuinely not be set.